portal/server/generateApp.ts

```diff
@@ -61,7 +68,7 @@ const generateApp = () => {
     };
   }
 
-  const redisClient = redis.createClient(process.env.REDIS_PORT, process.env.REDIS_HOSTNAME, redisOptions);
+  const redisClient = redis.createClient(parseInt(process.env.REDIS_PORT || '6379', 10), process.env.REDIS_HOSTNAME, redisOptions);
```

Comment:
`REDIS_PORT` will always exists in `env`, for fail-safe purpose if the port must be defined can it be centralised to a constant please and perhaps similar can be done for all the ports env variables.

Reply:
The other middleware, etc accept an `undefined` port but redis doesn't. I'll add a constant for `REDIS_PORT` but I think leaving the others to handle undefined env vars themselves is ok